subscription-manager.spec

```diff
@@ -187,6 +187,10 @@ Requires: %{?suse_version:dbus-1-python} %{!?suse_version:dbus-python}
 Requires: %{?suse_version:yum} %{!?suse_version:yum >= 3.2.29-73}
 %endif
 
+%if %{use_dnf}
+Requires: dnf-plugin-subscription-manager = %{version}
```

Comment:
This plugin should not be required on centos7/rhel7, because yum is used there and this change would add only unnecessary dependency.